### PR TITLE
Update 100-kotlin.md

### PR DESCRIPTION
**Pull request by SaktiSriraj**
Resources to learn Kotlin were added

### CODE REVIEW

File: src/data/roadmaps/android/content/100-pick-a-language/100-kotlin.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
...
\ No newline at end of file
+`Kotlin` is a cross-platform, statically typed general-purpose programming language with type inference. Developed by JetBrains, the makers of the world’s leading IDEs, Kotlin has a syntax, which is more expressive and concise. This allows for more readable and maintainable code. It is fully interoperable with Java and comes with no limitations. It can be used almost everywhere Java is used today, for server-side development, Android apps, and much more. Kotlin introduces several improvements for programmers over Java, which makes it a preferred choice for many developers. With more concise code base and modern programming concept support - it's certainly a future of Android app development.
+
+## Useful Resources
+
+[Learn Kotlin Programming for Beginners - Free Code Camp](https://youtu.be/EExSSotojVI?si=4VPW8ZHa2UMX0HH1) </br>
+[Learn Kotlin - w3schools](https://www.w3schools.com/kotlin/)
```
Comment (dansholds) [inline src/data/roadmaps/android/content/100-pick-a-language/100-kotlin.md:8]: Remove the line break and indent with a hyphen

 ```suggestion
- [Learn Kotlin Programming for Beginners - Free Code Camp](https://youtu.be/EExSSotojVI?si=4VPW8ZHa2UMX0HH1)
- [Learn Kotlin - w3schools](https://www.w3schools.com/kotlin/)
```

**Review by dansholds — CHANGES_REQUESTED** [review]
Good contribution overall but please remediate my comments.

In future please try to write a more descriptive PR title i.e. "Add this to that"

File: src/data/roadmaps/android/content/100-pick-a-language/100-kotlin.md (hunk 1 above)
Comment (SaktiSriraj) [reply src/data/roadmaps/android/content/100-pick-a-language/100-kotlin.md:8]: Okay. Will keep that in mind

File: src/data/roadmaps/android/content/100-pick-a-language/100-kotlin.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
\ No newline at end of file
+`Kotlin` is a cross-platform, statically typed general-purpose programming language with type inference. Developed by JetBrains, the makers of the world’s leading IDEs, Kotlin has a syntax, which is more expressive and concise. This allows for more readable and maintainable code. It is fully interoperable with Java and comes with no limitations. It can be used almost everywhere Java is used today, for server-side development, Android apps, and much more. Kotlin introduces several improvements for programmers over Java, which makes it a preferred choice for many developers. With more concise code base and modern programming concept support - it's certainly a future of Android app development.
+
+## Useful Resources
```
Comment (kamranahmedse) [inline src/data/roadmaps/android/content/100-pick-a-language/100-kotlin.md:5]:
```suggestion
Use following resources to learn more:
```

**Pull request merged**